<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresettingWindow</class>
 <widget class="QMainWindow" name="PresettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>20</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Виберіть тип гри:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameTypeGame">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>270</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonTwoPlayers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Два гравці</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonPlayPC">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Проти комп'ютера</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ButtonGoGame">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>260</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Грати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBack">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>300</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернутися назад</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameWhoFirst">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>270</width>
      <height>55</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>5</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Хто ходить перший:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonPlayerFirst">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Я</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonPCFirst">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Комп'ютер</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameDifficulty">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>190</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Рівень складності:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonEazy">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Легкий</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMiddle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Середній</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonHard">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Тяжкий</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
